test/specs/modules/Accordion/AccordionTitle-test.js

```diff
@@ -1,43 +1,9 @@
-import React from 'react'
 import * as common from 'test/specs/commonTests'
-import { sandbox } from 'test/utils'
 
-import Accordion from 'src/modules/Accordion/Accordion'
 import AccordionTitle from 'src/modules/Accordion/AccordionTitle'
 
 describe('AccordionTitle', () => {
   common.isConformant(AccordionTitle)
   common.rendersChildren(AccordionTitle)
   common.propKeyOnlyToClassName(AccordionTitle, 'active')
-
-  describe('onClick', () => {
```

Comment:
Intentional given that all the functionality is being tested by the event-related tests within `isConformant`